lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -1,34 +1,52 @@
+import 'package:pokedex_async_redux/features/pokemon_overview/widgets/pokemon_card.dart';
+import 'package:pokedex_async_redux/utilities/async.dart';
 import 'package:pokedex_async_redux/utilities/colors.dart';
 import 'package:pokedex_async_redux/utilities/constant.dart';
 import 'package:pokedex_async_redux/api/model/pokemon.dart';
-import 'package:pokedex_async_redux/features/pokemon_overview/widgets/pokemon_card.dart';
 import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/utilities/globals.dart';
 
 class PokemonOverviewPage extends StatelessWidget {
   const PokemonOverviewPage({
     required this.pokemons,
     Key? key,
   }) : super(key: key);
 
-  final List<Pokemon> pokemons;
+  final Async<List<Pokemon>> pokemons;
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: const Text(appName),
-        centerTitle: true,
-        backgroundColor: primaryColor,
-      ),
-      body: GridView.builder(
-        padding: const EdgeInsets.all(10.0),
-        itemCount: pokemons.length,
-        gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: pokemonColumnCount),
-        itemBuilder: (_, index) {
-          final pokemon = pokemons[index];
-          return PokemonCard(pokemon: pokemon);
-        },
+    return MaterialApp(
```

Comment:
why place your MaterialApp here? kindly revert your changes for this, Scaffold is enough 🙂